app/assets/stylesheets/sass/components/_pagination.scss

```diff
@@ -0,0 +1,47 @@
+.pagination {
+  $base-border-color: gainsboro !default;
+  $base-border-radius: 3px !default;
+  $base-spacing: 1.5em !default;
+  $pagination-border-color: $base-border-color;
+  $pagination-border: 1px solid $pagination-border-color;
+  $pagination-background: lighten($pagination-border-color, 10);
+  $pagination-hover-background: lighten($pagination-background, 5);
+  $pagination-color: $base-font-color;
+  text-align: center;
+
+  span.disabled {
+    display: none;
+  }
+
+  a {
+    display: inline;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered background, border, border-radius, color, display, margin, outline, padding, padding, text-align, text-decoration